static/lang/en.json

```diff
@@ -3345,7 +3345,7 @@
                     "Label": "Label"
                 },
                 "EquipmentTraits": {
-                    "Hint": "Traits applicable to equipment other than weapons",
+                    "Hint": "Traits applicable to equipment, armor, and consumables",
```

Comment:
shields, too, now? approaching "equipment other than weapons" :grin:

Reply:
nah, I made a separate one for shield. I thought about it a bit more and propagation works best when propagating from a child to a parent that are in a is-a relationship. Shield is kinda divorced and has too many unique traits.

Armor should probably also have been divorced originally, but I wasn't about to change it.